pinot-core/src/main/java/org/apache/pinot/core/data/table/Record.java

```diff
@@ -43,4 +46,9 @@ public Key getKey() {
   public Object[] getValues() {
     return _values;
   }
+
+  @Override
+  public String toString() {
+    return _key.toString() + " " + Arrays.deepToString(_values);
```

Comment:
Will this String be used for de-serialization? And can key have whitespaces?

Reply:
This String will never be used for de-serialization. We will always be using the granular values within the array for ser/deser.